systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -1089,6 +1089,257 @@ void waitForPodDeletion(String namespace, String podName) {
             () -> CLIENT.pods().inNamespace(namespace).withName(podName).get() == null);
     }
 
+    void verifyLabelsOnZkPods(String namespace, String clusterName, int zkPods, String appName) {
```

Comment:
@asorokhtey You can definitely parameterize similar methods. I can see there a lot of methods which are doing basically same thing and differs only in strings for pod names. Guess you can have 1 method for pods check, 1 for services, 1 for config maps and so on